src/components/ChangeOracles.tsx

```diff
@@ -0,0 +1,286 @@
+import { Menu, Transition } from '@headlessui/react';
+import clsx from 'clsx';
+import { AnimatePresence, motion } from 'framer-motion';
+import { LoadStatus, usePublishedDatum, WalletContext } from 'lib/wallet';
+import {
+  Fragment,
+  useCallback,
+  useContext,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { FiChevronDown, FiPlus, FiX } from 'react-icons/fi';
+import { bech32 } from 'bech32';
+
+interface ListItemProps {
+  address: string;
+  onRemove: () => void;
+}
+
+const ListItem = ({ address, onRemove }: ListItemProps) => {
+  return (
+    <div className="flex flex-row justify-between  text-purple-500 bg-purple-50 rounded-lg p-2">
+      <span className="leading-8">{address}</span>
+      <button
+        className="p-2 rounded hover:bg-purple-200"
+        type="button"
+        onClick={onRemove}
+      >
+        <FiX />
+      </button>
+    </div>
+  );
+};
+
+export enum ChangeOraclesMode {
+  Add = 0,
+  Remove = 1,
```

Comment:
equivalent:
```suggestion
  Add,
  Remove,
```
And since this mapping is only used in-memory we can reorder at will. i.e. the numbers don't matter so let's omit them.